FileHelpers/Engines/EngineBase.cs

```
@@ -189,6 +189,27 @@ public Encoding Encoding
 
         #endregion
 
+        #region "  NewLineForWrite  "
+        /// <summary>
+        /// Newline string to be used when engine writes to file. 
+        /// Default is the system's newline setting (System.Environment.NewLine).
+        /// </summary>
+        [DebuggerBrowsable(DebuggerBrowsableState.Never)]
+        protected String mNewLineForWrite = Environment.NewLine;
+
+        /// <summary>
+        /// Newline string to be used when engine writes to file. 
+        /// Default is the system's newline setting (System.Environment.NewLine).
+        /// </summary>
+        /// <value>Default is the system's newline setting.</value>
+        public string NewLineForWrite
+        {
+            get { return mNewLineForWrite; }
+            set { mNewLineForWrite = value; }

```

Good documentation!

I think the setter should have an argument check: At least setting _null_ should throw an exception, not sure if _string.empty_ should be allowed.